test/phoenix/digester_test.exs

```diff
@@ -0,0 +1,50 @@
+defmodule Phoenix.DigesterTest do
+  use ExUnit.Case, async: true
+
+  test "fails when the given paths are invalid" do
+    assert {:error, :invalid_path} = Phoenix.Digester.compile("../some invalid?/path", "/other_invalid ?? /path")
+  end
+
+  test "digests and compress files" do
+    output_path = Path.join("tmp", "compiled")
+    input_path = "priv/static"
+    File.mkdir(output_path)
+
+    assert :ok = Phoenix.Digester.compile(input_path, output_path)
+
+    [input_file, _]= assets_files(input_path)
```

Comment:
Let's pick one file explicitly? The test will be clearer if we have this:

```elixir
assert Enum.member?(output_files, "images/phoenix.png")
assert Enum.member?(output_files, "images/phoenix.png.gz")
```

and so on.